### PR TITLE
fixing the grammar error.

### PR DESCRIPTION
**Pull request by wyang2008**

### CODE REVIEW

**Comment by cydia2020** [comment]
Not Grammar Näzî enough my comrade.

File: 2019/04/2019-04-23-bilibili.md [hunk 1]
```diff
@@ -1,18 +1,16 @@
...
+Someone created a Repository called 'openbilibili/go-common' two hours ago and many people forked it or cloned it and reuploaded the source codes to new/different repositories, the repository contained too much sensitive information, such as secret key, database address, very important api key and so on. If these information are found by other people in Github, our company will be left out of pocket.
 
-The following is Github's URL and Screenshots about this  Repository:
+The following is Github's URL and Screenshots about this Repository:
 
 [private]
 
-Since the reupload repositories and the contains our company's important source code,algorithm and system configuration,,so we hope Giuhub as soon as possible to help us completely remove the reupload repositories and thank you very much
+Since the reupload repositories contain our company's important source code, algorithm and system configuration, so we hope Giuhub as soon as possible to help us completely remove the reupload repositories. Thank you very much!
```
Comment (wellzz) [inline 2019/04/2019-04-23-bilibili.md]: how about remove "Since"(or "so")

File: 2019/04/2019-04-23-bilibili.md [hunk 2]
```diff
@@ -1,18 +1,16 @@
 Hello Dear Github:
 
-  We Are:
-  
-  Bilibili Security Group.It is nice to be able to yse Github such a professional and effcient platform,and mpw we have a need to be very hopefully able to get Github's support and resolution.I sent the mail yesterday and the mail attachments is the my signature!!
+We Are Bilibili Security Group. It is nice to be able to use Github such a professional and effcient platform, and now we have a need and  hopefully we are able to get Github's support and resolution. I sent the mail yesterday and the mail attachment is my signature!!
 
-We have read and understand GitHub's Guide to Filling a DMCA Notice
+We have read and understand GitHub's Guide to Filling a DMCA Notice.
```
Comment (ihainan) [inline 2019/04/2019-04-23-bilibili.md:5]: understand -> understood.
to filling -> to fill.

File: 2019/04/2019-04-23-bilibili.md [hunk 3]
```diff
@@ -1,18 +1,16 @@
 Hello Dear Github:
```
Comment (ihainan) [inline 2019/04/2019-04-23-bilibili.md:1]: Github -> GitHub

File: 2019/04/2019-04-23-bilibili.md (hunk 2 above)
Comment (cydia2020) [reply 2019/04/2019-04-23-bilibili.md:5]: It's actually to file

**Comment by jessephus** [comment]
Is fixing grammar in Bilibili notices some sort of new Internet meme? 😄 

I assume this is just a joke. But, if not, we just re-post the text we receive in DMCA notices here. Because of that, we [do not accept contributions](https://github.com/github/dmca/blob/master/CONTRIBUTING.md) to change or fix notices.

**Pull request closed without merging**